### DISCUSSION TITLE
五月天假唱...

### DISCUSSION THREAD

**Discussion by spermwhale464** [1101-GS4719-C/Python-Community-GS4719-C#1770] [General]
五月天假唱近期鬧的水生火熱，作為從初中就開始聽五月天的粉絲，看到很多網友用了程式跑聲音對比抑或是音準辦別，得出的結果都跟MP3的音檔近乎相符，不禁有點失落，當年音樂界的神壇就此殞落嗎.......要問問大家都怎麼看這件事?

**Comment by Terryyyyyyyyyyyy** [comment]
我認為要以比較平靜的心態去看吧，我認為適時的添加墊音或者其他輔助歌手的方式，是可以的，但他僅限用於增加演唱效果，而不是改善整體的聲音等。
放錄音帶然後對嘴是可恥的，歌手本來就是以現場演唱為工作，結果你直接讓歌迷花錢進場聽cd。

**Comment by Ryanlolry** [comment]
我認為五月天其實可以學五百的方式，如果嗓音狀態不好，可以選擇和歌迷互動，而他們在事後處理也不太合理，有嚴重聲明的方式痛斥假唱，且加開直播證明，但反正更加實錘他們假唱的事實，我認為就大方承認吧。
嗓音狀況不好有其他方式，如薛之謙選擇退票，或梁靜茹堅持真唱，破音後歌迷也是認為他狀況不好，但其實他們是可以體諒的，而且他也透過後續的演唱會證明他的嗓音僅在那場出現一點失誤，再之後也鮮有他的負面消息。
五月天現在把自己推到一個不可挽回的地步了。

**Comment by Yanxxx7979** [comment]
假唱後的處理態度的確會令人心寒吧
但我個人認為他們能走到現在，更多的是膾炙人口的曲子
畢竟他們是擁有作曲能力的樂隊
如果出來新歌有中的話感覺歌迷還是會買單

**Comment by davidteng00000** [comment]
網路上有一些相關案例是歌手也不知道自己在假唱，因為歌手被設計了，不能一昧的去責怪歌手，我們應該也要去審視一下相關的公司。不過假唱真的是十分不道德的行為，欺騙消費者與紛絲

**Comment by uwje** [comment]
我認為歌手如果要假唱還不如就不要唱，畢竟歌迷去演唱會是要享受那個環境的，結果你放個錄音帶在對嘴，要不就承認當天狀況不好然後盡全力唱，要不然就退票。

**Comment by yuchien-lin** [comment]
我認為一個這麼有名的樂團不可能假唱，如果有也不是他們自願，像是周杰倫從幾年前的巡迴演唱會以後聲帶就受損了，他一開演唱會還是會每場票都售空，所以如果五月天假唱的話那為了是什麼，怕別人知道他無法唱的跟原本一樣嗎，可是從周杰倫的演唱會來看，不論他演唱的表現如何，也能受到粉絲的支持，所以假唱對他們來說完全只有弊，一個這麼有名的樂團怎麼可能做這種事。

**Comment by woohsin** [comment]
我對此事沒什麼意見，但我覺得大概是有假唱啦~重點是我媽是五迷，所以我只要跟我媽聽到五月天的歌我就會跟他說假唱阿信，然後他就會瞪我，雖然有點白目，但是我媽的反應真的很可愛。

**Comment by benbebobo149** [comment]
我覺得那是五月天主唱自尊心問題，他應該是衡量這些風險之後才做出這樣的決定。但對歌迷來說是一個蠻不負責任的行為，只顧到自己的利益，而非以粉絲為優先出發。雖然我覺得我們不需要太可憐他，毕竟這是他的個人選擇，但我認為藝人與粉絲之間應該是相互尊重和理解的關係。無論如何，我認為這樣的事情還是應該由市場和粉絲的支持來決定，而不是由單一個人做出決定。最後，我要感謝同學的分享，這讓我更深入地思考了藝人與粉絲之間的關係和互動。
